[2024-01-31T15:22:21.074+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cvd_automation_pipeline.run_nuclei_scan_task manual__2024-01-31T15:22:04.982374+00:00 [queued]>
[2024-01-31T15:22:21.081+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cvd_automation_pipeline.run_nuclei_scan_task manual__2024-01-31T15:22:04.982374+00:00 [queued]>
[2024-01-31T15:22:21.082+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-01-31T15:22:21.094+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): run_nuclei_scan_task> on 2024-01-31 15:22:04.982374+00:00
[2024-01-31T15:22:21.098+0000] {standard_task_runner.py:60} INFO - Started process 15228 to run task
[2024-01-31T15:22:21.100+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'cvd_automation_pipeline', 'run_nuclei_scan_task', 'manual__2024-01-31T15:22:04.982374+00:00', '--job-id', '33', '--raw', '--subdir', 'DAGS_FOLDER/cvd_pipeline_1.py', '--cfg-path', '/tmp/tmpl0j23rpf']
[2024-01-31T15:22:21.102+0000] {standard_task_runner.py:88} INFO - Job 33: Subtask run_nuclei_scan_task
[2024-01-31T15:22:21.134+0000] {task_command.py:423} INFO - Running <TaskInstance: cvd_automation_pipeline.run_nuclei_scan_task manual__2024-01-31T15:22:04.982374+00:00 [running]> on host 8d8bbdf320cd
[2024-01-31T15:22:21.199+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='cvd_automation_pipeline' AIRFLOW_CTX_TASK_ID='run_nuclei_scan_task' AIRFLOW_CTX_EXECUTION_DATE='2024-01-31T15:22:04.982374+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-31T15:22:04.982374+00:00'
[2024-01-31T15:22:21.900+0000] {logging_mixin.py:188} INFO - An error occurred while running Nuclei command: Command 'nuclei -H 'User-Agent: DIVD-2023-00020' -t /opt/***/config/fingerprint_files/fingerprint_data.yaml -l data/DIVD-2024-XXXX/2024-01-31-shodan-results.csv -retries 2 -timeout 6 -output data/DIVD-2024-XXXX/2024-01-31-nuclei-results.json -j' returned non-zero exit status 1.
[2024-01-31T15:22:21.901+0000] {logging_mixin.py:188} INFO - None
[2024-01-31T15:22:21.903+0000] {logging_mixin.py:188} INFO - Nuclei scan completed.
[2024-01-31T15:22:21.904+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-01-31T15:22:21.912+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=cvd_automation_pipeline, task_id=run_nuclei_scan_task, execution_date=20240131T152204, start_date=20240131T152221, end_date=20240131T152221
[2024-01-31T15:22:21.957+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-31T15:22:21.971+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
